scripts/migration/sanityCheck.sql

```diff
@@ -0,0 +1,49 @@
+﻿-- report values taken from https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/821334/Tuberculosis_in_England-annual_report_2019.pdf
```

Comment:
Should this file be here? seems like smth you used for testing

Reply:
It is, but it's also something that might just come in handy again, so I'd rather keep it checked in. In that sense it's similar to the javascript analysis snippets in the same folder!